pkgs/development/python-modules/manim/default.nix

```diff
@@ -106,15 +130,24 @@ in python3.pkgs.buildPythonApplication rec {
     ])
   ];
 
-
   nativeCheckInputs = [
-    python3.pkgs.pytest-xdist
-    python3.pkgs.pytestCheckHook
+    pytest-xdist
+    pytestCheckHook
 
     ffmpeg
     (texlive.combine manim-tinytex)
+    xorg.xorgserver
   ];
 
+  # fails like glcontext:
+  # Exception: (standalone) glXChooseFBConfig failed
+  doCheck = false;
```

Comment:
Could we disable individual tests (see line 152 and `failing_tests.nix`) instead of disabling all testing?

Reply:
Currently tests fail in initialization, so no tests can be run